lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -358,6 +365,33 @@ def absent_strategy(compute_api, wished_server):
     return changed, {"status": "Server %s deleted" % target_server["id"]}
 
 
+def terminate_strategy(compute_api, wished_server):
+    compute_api.module.debug("Starting terminate strategy")
+    changed = False
+    target_server = None
+    query_results = find(compute_api=compute_api, wished_server=wished_server, per_page=1)
+
+    if not query_results:
+        return changed, {"status": "Server already terminated."}
+    else:
+        target_server = query_results[0]
+
+    changed = True
+
+    if compute_api.module.check_mode:
+        return changed, {"status": "Server %s would be made terminated." % target_server["id"]}
+
+    # Here we should check that the server is running and not locked.
+    # Terminate a server WITHOUT snapshot before
+    response = terminate_server(compute_api=compute_api, server=target_server,)
+
+    if not response.ok:
+        err_msg = 'Error while removing server [{0}: {1}]'.format(response.status_code, response.json)
```

Comment:
So, are we sure the response is going to be json ? Like, if the server fail to answer ?